HISTORY.md

```diff
@@ -828,59 +828,59 @@
 * chore(common/resources): bump @actions/http-client from 1.0.3 to 1.0.8 in /resources/build/version (#3047)
 * fix(oem/fv/ios): firstvoices icon and version info (#3044)
 * feat(windows): Sentry integration fixes and polish (#3006)
-* feat(common/lmlayer): compile joinWordsAt property (#3032)
+* feat(developer/compilers): compile joinWordsAt property (#3032)
 
 ## 14.0.60 alpha 2020-04-29
 
-* feat(common): initial use of lerna (in-repo package links only) (#2997)
+* feat(common/resources): initial use of lerna (in-repo package links only) (#2997)
```

Comment:
This one reflects why I consider the previous `common/resources` change to be debatable.  This one _is_ closer to `common/resources`, as it does _not_ directly affect build products; the changes at this point were development-targetted only.  I don't mind leaving this one as-is.